packages/core/package.json

```diff
@@ -3,7 +3,10 @@
   "version": "4.2.0",
   "description": "A simple React component capable of building HTML forms out of a JSON schema.",
   "scripts": {
-    "build": "rimraf dist lib && npx tsc && tsdx build --format cjs,es && cross-env NODE_ENV=production BABEL_ENV=umd webpack --config webpack.config.dist.js",
+    "build": "rimraf dist && tsdx build --format cjs,es && cross-env NODE_ENV=production BABEL_ENV=umd webpack --config webpack.config.dist.js && npm run deprecated:build:cjs && npm run deprecated:build:es && npm run deprecated:build:es:lib",
```

Comment:
Do we even need the tsdx build now?

Reply:
I'm worried that removing it will create a breaking change from v4.2 to this version

Reply:
ah, ok